cmd/query/app/handler.go

```diff
@@ -47,6 +47,9 @@ const (
 
 	defaultDependencyLookbackDuration = time.Hour * 24
 	defaultTraceQueryLookbackDuration = time.Hour * 24 * 2
+
+	// DefaultHTTPPrefix is the prefix for the API if no prefix is provided
+	DefaultHTTPPrefix = "api"
```

Comment:
for what purpose? It's already defaulted here, doesn't make sense to re-default. Unless you want to add some other routes to the `/api` prefix, which I don't think is a good idea, extensions should have their own paths.